server/auth/store.go

```diff
@@ -216,14 +218,18 @@ type AuthBatchTx interface {
 }
 
 type AuthReadTx interface {
+	RLock()
+	RUnlock()
+	UnsafeReader
+}
+
+type UnsafeReader interface {
```

Comment:
```suggestion
type UnsafeAuthReader interface {
```

Reply:
It's in package auth so `backend.UnsafeReader` and `auth.UnsafeReader` should suffice. If not I can link you this convention in https://go.dev/doc/effective_go

Reply:
https://go.dev/blog/package-names

Reply:
`AuthReadTx` and `AuthBatchTx` are all bad names, the point is to keep them consistent. Also when we search the code base using "UnsafeReader", so as to differentiate with the "UnsafeReader" in package backend.

I won't insist on this. Probably we should correct `AuthReadTx` and `AuthBatchTx` as well.

Reply:
I can fix that, if you have review capacity.

Reply:
We already have `AuthBackend`, `AuthBatchTx`, `AuthReadTx` and `UnsafeAuthReader` under package `auth`.   Although they are all bad names based on golang's convention, but it's clearer from pragmatic perspective because it's clearly differentiated from `Backend`, `BatchTx`, `ReadTx` and `UnsafeReader` under package backend.

Reply:
Ok, don't care. Let's just merge it.